### PR TITLE
Removed "tags:" frontmatter metadata

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

- [Removed `tags:` frontmatter metadata](https://sumologic.atlassian.net/browse/DOCS-253)
   - NOTE: I will remove `tags` from C2C docs in a separate PR, as those may contain automation. John will remove `tags` from App Central docs in a separate PR.
- Other misc. minor formatting/wording fixes

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [x] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: blog-cse/2024-02-19-content.md [hunk 1]
```diff
@@ -24,4 +23,4 @@ This release includes new log mapping and parsing content for Druva Cyber Resili
 
 #### Bug Fixes
 
-* Recently, two rules, FIRST-S00052 and FIRST-S00049, were released to customers erroneously. Soon after, these rules started generating false positive Signals and Insights. We have removed those rules from all customer environments so they can be tuned properly and re-released after comprehensive testing. The process error that led to the release has been identified and corrected. Sumo Logic apologizes for the inadvertent Signals and Insights this error generated. If needed, please contact Support for assistance in closing the Insights. 
\ No newline at end of file
+* Recently, two rules, FIRST-S00052 and FIRST-S00049, were released to customers erroneously. Soon after, these rules started generating false positive Signals and Insights. We have removed those rules from all customer environments so they can be tuned properly and re-released after comprehensive testing. The process error that led to the release has been identified and corrected. Sumo Logic apologizes for the inadvertent Signals and Insights this error generated. If needed, please contact Support for assistance in closing the Insights.
```
Comment (kimsauce) [inline blog-cse/2024-02-19-content.md:26]:
```suggestion
* Recently, two rules, FIRST-S00052 and FIRST-S00049, were released to customers erroneously. Soon after, these rules started generating false positive Signals and Insights. We have removed those rules from all customer environments so they can be tuned properly and re-released after comprehensive testing. The process error that led to the release has been identified and corrected. Sumo Logic apologizes for the inadvertent Signals and Insights this error generated. If needed, please contact Support for assistance in closing the Insights. 
```

File: blog-cse/2024-02-23-content.md [hunk 2]
```diff
@@ -16,7 +15,7 @@ This content release includes modifications and additions to Citrix Cloud C2C to
 
 #### Log Mappers
 
-* [Deleted] Citrix Cloud Client 
+* [Deleted] Citrix Cloud Client
```
Comment (kimsauce) [inline blog-cse/2024-02-23-content.md:18]:
```suggestion
* [Deleted] Citrix Cloud Client 
```

File: docs/get-started/quickstart.md [hunk 3]
```diff
@@ -48,7 +49,7 @@ OpenTelemetry is our preferred collector unless you're configuring security data
 #### Installed and Hosted Collector
 
 1. Log in to Sumo Logic.
-1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Collection > Collection**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu select **Configuration**, and then under **Data Collection** select **Collection**. You can also click the **Go To...** menu at the top of the screen and select **Collection**. Kanso--> 
+1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Collection > Collection**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu select **Configuration**, and then under **Data Collection** select **Collection**. You can also click the **Go To...** menu at the top of the screen and select **Collection**. Kanso-->
```
Comment (kimsauce) [inline docs/get-started/quickstart.md:52]:
```suggestion
1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Collection > Collection**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu select **Configuration**, and then under **Data Collection** select **Collection**. You can also click the **Go To...** menu at the top of the screen and select **Collection**. Kanso--> 
```

**Review by jpipkin1 — APPROVED** [review]
Thank you for removing these tags, @kimsauce.

**Pull request merged**